build.sh

```diff
@@ -15,6 +15,9 @@ rm -rf pkg/*
 rm -rf $GOPATH/pkg/*
 mkdir -p bin/
 
+# Fix for build failing due to missing go-vnc
+(cd $GOPATH/src/github.com/mitchellh && if cd go-vnc; then git pull; else git clone https://github.com/mitchellh/go-vnc.git; fi)
```

Comment:
I'm not sure why this is required? the repo should be able to be vendored like the rest. Did you try this?